public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,188 @@
+  project's root.
+
+- `repo` - specifies the location of the DVC project. It can be a URL or a file
+  system path. Both HTTP and SSH protocols are supported for online Git repos
+  (e.g. `[user@]server:project.git`). _Default_: The current project is used
+  (the current working directory tree is walked up to find it).
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
+  option is ignored. _Default_: `HEAD`.
+
+- `remote` - name of the [DVC remote](/doc/command-reference/remote) to look for
+  the target data. _Default_: The
+  [default remote](/doc/command-reference/remote/default) of `repo` is used if a
+  `remote` argument is not given. For local projects, the <abbr>cache</abbr> is
+  tied before the default remote.
+
+- `mode` - specifies the mode in which the file is opened. Defaults to `"r"`
+  (read). Mirrors the namesake parameter in builtin
+  [`open()`](https://docs.python.org/3/library/functions.html#open).
+
+- `encoding` -
+  [codec](https://docs.python.org/3/library/codecs.html#standard-encodings) used
+  to decode the file contents to a string. This should only be used in text
+  mode. Defaults to `"utf-8"`. Mirrors the namesake parameter in builtin
+  `open()`.
+
+## Exceptions
+
+- `dvc.exceptions.FileMissingError` - file in `path` is missing from `repo`.
+
+- `dvc.exceptions.PathMissingError` - `path` cannot be found in `repo`.
+
+- `dvc.api.UrlNotDvcRepoError` - `repo` is not a DVC project.
+
+- `dvc.exceptions.NoRemoteError` - no `remote` is found.
+
+## Example: Use data tracked in a DVC repository online
+
+Any <abbr>data artifact</abbr> can be employed directly in your Python app by
+using this API.
+
+For example, an XML file from a public DVC repo online can be processed directly
+in your Python app with:
+
+```py
+from xml.dom.minidom import parse
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    xmldom = parse(fd)
+    # ... Process DOM
+```
+
+> Notice that you could read the contents of a tracked file faster with
+> `dvc.api.read()`:
+>
+> ```py
+> xmldata = dvc.api.read('get-started/data.xml',
+>     repo='https://github.com/iterative/dataset-registry')
+> xmldom = parse(xmldata)
+> ```
+
+Now let's imagine you want to deserialize and use a binary model from a private
+repo online. For a case like this, we can use a SSH URL instead (assuming the
+[credentials are configured](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh)
+locally):
+
+```py
+import pickle
+import dvc.api
+
+with dvc.api.open(
+        'model.pkl',
+        repo='git@server.com:path/to/repo.git'
+        ) as fd:
+    model = pickle.load(fd)
+    # ... Use instanciated model
+```
+
+## Example: Use other versions of data or results
```

Comment:
other -> specific?

Reply:
... of a data file or model? ... results - we never used this term

Reply:
Updated.

> p.s. we do use "results" (mostly "intermediate results") a lot, but not in this doc.